### ISSUE TITLE
[StructuralMechanicsApplication] missing rotational dofs for 3D in GetDofList in base_load_condition.cpp

### ISSUE THREAD

**Issue by aronnoordam** [KratosMultiphysics/Kratos#11844]
missing rotational dofs for 3D in GetDofList in base_load_condition.cpp

**Comment by loumalouomega** [comment]
This is a request or a TODO?

**Comment by aronnoordam** [comment]
Just a report, I don't need it, I just saw it. But I can fix it

**Comment by loumalouomega** [comment]
As you want, but base load condition only required ROTATION when the derived one considers rotations, maybe that it is set on the derived class.

**Comment by aronnoordam** [comment]
but is inconsistent with the 2D case, where it does check if rot DOFS are present and then adds the Rotation if required

**Comment by loumalouomega** [comment]
> but is inconsistent with the 2D case, where it does check if rot DOFS are present and then adds the Rotation if required

You are right, feel free to fix i. right now I have several things on my side.
